tests/manage/OCS_336_346.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import pytest
+
+from ocs import ocp, defaults
+from ocsci.config import ENV_DATA
+from ocsci.testlib import tier1, ManageTest
+from resources.ocs import OCS
+from resources.pod import get_admin_key_from_ceph_tools
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+POD = ocp.OCP(kind='Pod', namespace=ENV_DATA['cluster_namespace'])
+CEPH_OBJ = None
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Create disks
+    """
+    self = request.node.cls
+    setup(self)
+    yield
+    teardown()
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    global CEPH_OBJ
+    self.fs_data = defaults.CEPHFILESYSTEM_DICT.copy()
+    self.fs_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'cephfs'
+    )
+    self.fs_data['metadata']['namespace'] = ENV_DATA['cluster_namespace']
+    CEPH_OBJ = OCS(**self.fs_data)
+    CEPH_OBJ.create()
+    assert POD.wait_for_resource(
+        condition='Running', selector='app=rook-ceph-mds'
+    )
+    pods = POD.get(selector='app=rook-ceph-mds')['items']
+    assert len(pods) == 2
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    global CEPH_OBJ
+    CEPH_OBJ.delete()
+
+
+@tier1
+class TestOSCBasics(ManageTest):
+    mons = """
+        rook-ceph-mon-a.rook-ceph.svc.cluster.local:6789,
+        rook-ceph-mon-b.rook-ceph.svc.cluster.local:6789,
+        rook-ceph-mon-c.rook-ceph.svc.cluster.local:6789
+    """
+
+    def test_ocs_336(self, test_fixture):
+        """
+        Testing basics: secret creation, storage class creation with cephfs
+        """
+        self.cephfs_secret = defaults.CEPHFS_SECRET.copy()
+        del self.cephfs_secret['data']['userID']
+        del self.cephfs_secret['data']['userKey']
+        self.cephfs_secret['data']['adminKey'] = (
+            get_admin_key_from_ceph_tools()
+        )
+        self.cephfs_secret['data']['adminID'] = defaults.ADMIN_BASE64
+        logging.info(self.cephfs_secret)
+        secret = OCS(**self.cephfs_secret)
+        assert secret.create()
```

Comment:
All the asserts will fail on running ocp functions

Reply:
Removed all assert checkings